src/main/java/io/xpring/payid/PayIDDiscoveryExceptionType.java

```diff
@@ -0,0 +1,12 @@
+package io.xpring.payid;
+
+/**
+ * Types of {@link PayIDDiscoveryException}.
+ */
+@SuppressWarnings("checkstyle:AbbreviationAsWordInName")
+public enum PayIDDiscoveryExceptionType {
+  INVALID_PAY_ID,
```

Comment:
nit: Alphabetize your enum cases? Unless you explicitly have some other ordering here.